ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/button/v1/button/clientlibs/site/js/buttonview.js

```diff
@@ -48,15 +51,15 @@
          * @returns {HTMLElement}
          */
         getDescription() {
-            return null;
+            return this.element.parentNode ? this.element.parentNode.querySelector(Button.selectors.description) : null;
         }
 
         getErrorDiv() {
             return null;
         }
 
         getTooltipDiv() {
-            return null;
+            return this.element.parentNode ? this.element.parentNode.querySelector(Button.selectors.tooltipDiv) : null;
```

Comment:
this.element should refer to the top level div element.

Reply:
If we shift this.element to parent div, then onclick event is getting triggered when the div is clicked.